### DISCUSSION TITLE
Should Plot cheatsheet include sliders for padding?

### DISCUSSION THREAD

**Discussion by harrylojames** [observablehq/plot#1736] [Ideas]
To add empty space to the top and keep the y label fixed to the axis you appear to need to use paddingTop. Thoughts on making it clear that this option exists in the cheatsheet? I can see that one might not want to present too many options too quickly to new users. 

https://observablehq.com/@observablehq/plot-cheatsheets-layouts#layoutsExample

https://observablehq.com/d/f1c92057a24612b0

This y label position 
![image](https://github.com/observablehq/plot/assets/51630653/b3e440b8-f005-473c-9072-0e28c8fc1ff1)

And not this y label position 
![image](https://github.com/observablehq/plot/assets/51630653/4b57c0e8-71b0-4a23-8138-fc91a25a700b)

**Comment by Fil** [comment]
The cheatsheet given as a reference is not maintained anymore (I'm even surprised you still had access to it… I don't)

**Reply by harrylojames** [reply]
Good to know thank you. I stumbled on it via google.
